### PR TITLE
Onboarding auth touchups

### PR DESCRIPTION
**Pull request by JimmyMcBride**
# Description

Fixes # (issue)
No yellow or red errors in console that I could find. Everything seems to be resolved! Yay!

## Type of change
Touched up code, everybody branches have been merged in and I wanted to do a little clean up!

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>NutriJournal</title>
```
Comment (israelgonzalezb) [inline public/index.html:27]: This makes sense

File: src/components/Onboarding/components/WeightGoal.js [hunk 2]
```diff
@@ -25,13 +25,18 @@ class WeightGoal extends React.Component {
...
     });
-    this.props.history.push("/register");
+    if (target_weight) {
+      this.props.history.push("/register");
+    } else {
+      alert("You need to fill out your target weight to continue.");
+    }
   };
 
   // This function can handle change, can you?
```
Comment (israelgonzalezb) [inline src/components/Onboarding/components/WeightGoal.js:42]: I used to be like two feet tall and I dealt with that ok

File: public/index.html (hunk 1 above)
Comment (dave-vazquez) [reply public/index.html:27]: true

**Review by israelgonzalezb — APPROVED** [review]
Lookin' good!

File: src/App.js [hunk 3]
```diff
@@ -26,30 +26,55 @@ import withNavigation from "./components/Navigation/withNavigation";
...
+  iconColor: "black",
+  topNavColor: "white",
+  displayBottom: false
+})(Register);
+
+const DailyLogWithNav = withNavigation({
+  pageTitle: "Daily Log"
+})(DailyLog);
+
+const FoodItemWithNav = withNavigation({
```
Comment (dave-vazquez) [inline src/App.js:60]: We gotta drill a little deeper into FoodItem to get the Navigation working (unless it's working on your branch). I got this one working on the daily-log branch tho, so no biggy.

Comment (JimmyMcBride) [reply src/App.js:60]: I got it working on my branch as well

**Review by chazr88 — APPROVED** [review]
I dont see any issues. Looks good to me!

**Review by dave-vazquez — APPROVED** [review]

File: src/App.js (hunk 3 above)
Comment (JimmyMcBride) [reply src/App.js:60]: I wrapped FoodItem with Container instead of a <> and set the prop height and it's working well

Comment (dave-vazquez) [reply src/App.js:60]: dope, sounds good

**Pull request merged**